app.js

```diff
@@ -49,7 +49,7 @@ app.use((err, req, res) => {
   res.status(err.status || 500);
   res.render("error", {
     message: err.message,
-    error: {}
+    error: {},
```

Comment:
Can you remove code style changes?

Reply:
These style changes were required by the linter. Please see the failing CI on prior commit.

Reply:
Oh. Thanks. I didn’t updater linter config yet (spend too much time on preparing PostCSS 8 😓).